src/lucky/tags/custom_tags.cr

```diff
@@ -32,4 +32,10 @@ module Lucky::CustomTags
     yield
     @view << "</#{name}>"
   end
+
+  def inline_tag(name : String, options = EMPTY_HTML_ATTRS, **other_options)
```

Comment:
What do you think of calling this empty_tag? I think they are usually referred to as empty/void tags https://www.456bereastreet.com/archive/201005/void_empty_elements_and_self-closing_start_tags_in_html/